Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -455,5 +455,5 @@ bool VertexLoaderX64::IsInitialized()
 int VertexLoaderX64::RunVertices(DataReader src, DataReader dst, int count, int primitive)
 {
 	m_numLoadedVertices += count;
-	return ((int (*)(u8* src, u8* dst, int count))region)(src.GetPointer(), dst.GetPointer(), count);
+	return ((int(*)(u8* src, u8* dst, int count))region)(src.GetPointer(), dst.GetPointer(), count);
```

Comment:
This space is actually on purpose because it separates the type from the function pointer asterisk.